deploy/charts/google-cas-issuer/Chart.yaml

```diff
@@ -12,5 +12,5 @@ maintainers:
 sources:
 - https://github.com/jetstack/google-cas-issuer
 
-appVersion: v0.6.2
-version: v0.6.2
+appVersion: v0.6.3
+version: v0.6.3
```

Comment:
There's already a 0.7.0 release, so should this be 0.7.1 or 0.8.0 ?
 * https://github.com/jetstack/google-cas-issuer/releases/tag/v0.7.0

Reply:
Oh... I hadn't actually clocked that all of this is overriden by the release action and hidden away in there 😭 

Quite different to our other release processes but I'd expected it to be similar. Maybe I misremembered it!

Never mind, then - this PR is entirely waste. I'll close! Thanks for taking a look!